src/views/penalty-details.njk

```diff
@@ -66,10 +66,15 @@
         }}
         <div class="govuk-form-group">
           {% set detailsHTML %}
-          <p>
-            Your company number and reference number are shown at the bottom of your penalty notice.
-          </p>
-          <img src="//{{ cdn.host }}/images/late-filing-penalties/penalty-notice.png" class="call-out-image" alt=""/>
+            <p>
+              Some penalty notices will show the company number and reference number at the bottom.
+              The penalty reference will be in the format A00000000
```

Comment:
I know that you've copied this word for word, but should there be a full stop at the end of this?